scripts/combine/setupCombine.py

```diff
@@ -182,26 +185,32 @@ def setup(args, inputFile, fitvar, xnorm=False):
         # FIXME: at some point we should decide what name to use
         if any(x in args.excludeProcGroups for x in ["BkgWmunu", "outAccWmunu"]) and hasSeparateOutOfAcceptanceSignal:
             datagroups.deleteGroup("BkgWmunu") # remove out of acceptance signal
-    else:
-        if "BkgWmunu" in args.excludeProcGroups:
+    elif "BkgWmunu" in args.excludeProcGroups:
             datagroups.deleteGroup("Wmunu") # remove out of acceptance signal
 
-    # Start to create the CardTool object, customizing everything
-    cardTool = CardTool.CardTool(xnorm=xnorm, ABCD=wmass and args.ABCD)
     cardTool.setDatagroups(datagroups)
+    cardTool.addProcessGroup("single_v_samples", lambda x: x[0] in ["W", "Z"] and x[1] not in ["W","Z"])
```

Comment:
These process groups are added further down again

Reply:
You mean it's duplicated or you mean this order causes issues? I don't think it's duplicated, and it passes the CI

Reply:
I mean it's duplicated, it appears again in L288